lookout/style/typos/config.py

```diff
@@ -28,6 +28,7 @@
         "path": str(DEFAULT_DATA_DIR / "fasttext.bin"),  # Where to store trained fasttext model
         "dim": 10,  # Number of dimensions of embeddings
         "bucket": 200000,  # Number of hash buckets in the model
+        "adjust_frequencies": True,
```

Comment:
```suggestion
        "adjust_frequencies": True,  # Whether to divide identifiers frequencies by tokens number.
```
Please adjust the comment if I get it wrongly.